qt/applications/workbench/workbench/plotting/figurewindow.py

```diff
@@ -19,6 +20,33 @@
 # local imports
 from mantidqt.plotting.figuretype import FigureType, figure_type
 from mantidqt.widgets.observers.observing_view import ObservingView
+from mantid.api import AnalysisDataServiceImpl
+import mantid.kernel
+
+
+def _validate_workspaces(names: List[str]) -> List[bool]:
+    """
+    Checks that workspaces being over plot have multiple bins
+    :param names: A list of workspace names
+    :return: List of bools, True if all workspaces have multiple bins
+    """
+    ads = AnalysisDataServiceImpl.Instance()
+    results = []
+    for name in names:
+        result = False
```

Comment:
Could maybe call it "has_multiple_bins" or something similar, instead of "result"?